docker_test.go

```diff
@@ -279,6 +279,27 @@ func TestContainerStateAfterTermination(t *testing.T) {
 		require.Nil(t, state, "expected nil container inspect.")
 	})
 
+	t.Run("Nil State after termination with timeout", func(t *testing.T) {
```

Comment:
suggestion: we're standardising on short names with no spaces so a search from the failed sub test name can be successful.
```suggestion
	t.Run("termination-timeout", func(t *testing.T) {
```